internal/controller/dpuoperatorconfig_controller.go

```diff
@@ -120,7 +120,7 @@ func (r *DpuOperatorConfigReconciler) createCommonData(cfg *configv1.DpuOperator
 	data := map[string]string{
 		"Namespace":              "openshift-dpu-operator",
 		"ImagePullPolicy":        r.imagePullPolicy,
-		"Mode":                   cfg.Spec.Mode,
+		"Mode":                   "auto",
```

Comment:
Is this intended to be hardcoded? Or left over from debug?

Reply:
intented.